test/test-utils.ts

```diff
@@ -55,14 +55,14 @@ function activateVirtualenv(venvName) {
 function deactivateVirtualenv() {
   if (getActiveVenvName() === null) {
     console.warn('Attempted to deactivate a virtualenv when none was active.');
-    return;
+    return () => {};
```

Comment:
why is this needed? Does it affect what we expect to receive on the caller side?

Reply:
For typescript, for types to match. This function needs to return a cleanup function.